spec/lib/topic_query_spec.rb

```diff
@@ -1940,4 +1940,29 @@ def suggested_for(topic)
       ).to contain_exactly(category.topic_id, subcategory.topic_id, subcategory_regular_topic.id)
     end
   end
+
+  describe "with topic_query_create_list_topics modifier" do
+    let!(:topic1) { Fabricate(:topic, created_at: 3.days.ago, bumped_at: 1.hour.ago) }
+    let!(:topic2) { Fabricate(:topic, created_at: 2.days.ago, bumped_at: 3.hour.ago) }
```

Comment:
Can `fab!` be used here? (as a rule of thumb it's better to stick to it whenever possible)

```suggestion
    fab!(:topic1) { Fabricate(:topic, created_at: 3.days.ago, bumped_at: 1.hour.ago) }
    fab!(:topic2) { Fabricate(:topic, created_at: 2.days.ago, bumped_at: 3.hour.ago) }
```